distributed/__init__.py

```diff
@@ -1,5 +1,10 @@
-from distributed import config  # isort:skip; load distributed configuration first
-from distributed import widgets  # isort:skip; load distributed widgets second
+from __future__ import annotations
+
+# isort: off
```

Comment:
`isort: skip` doesn't work unless the block of skipped imports is at the top or bottom of all the imports

Reply:
https://github.com/PyCQA/isort/issues/1337